server/controller/TestDataController.ts

```diff
@@ -1,10 +1,11 @@
 import { NextFunction, Request, Response } from "express";
-
 import express from "express";
 import logger from "../config/logger";
 import { deleteCache } from "./CacheManager";
 import db from "../db";
 import { findLatestForToUser } from "../test/testEmailCache";
+import tk from "timekeeper";
```

Comment:
Gibts keinen anderen Weg außer ein package, welches seit drei Jahren nicht aktualisiert wurde?

Reply:
Das hat immer hin noch 150.000 Downloads und die Issues türmen sich auch nicht auf :-D
Das sah von der API am einfachsten aus.

Gerade nochmal auf NPM gewühlt. Es gäbe noch mockdate. Das ist "nur" 2 Jahre  und hat sogar 1.500.000 Downloads. 

Würde aber tatsächlich lieber bei der aktuellen Lösung bleiben.